dotcom-rendering/src/components/Card/Card.tsx

```diff
@@ -459,7 +459,6 @@ export const Card = ({
 										format={format}
 										isMainMedia={true}
 										hideCaption={true}
-										role="inline"
```

Comment:
We actually never use any role other than "inline" in DCR for the YoutubeAtom. I've deleted a lot of the code around deciding the best image size based on role type in this PR.

Reply:
Yeah, super-sadly and absurdly, one can weigh custom embed in Composer, but not atoms. We will get to [this card](https://trello.com/c/yeFnbhkO/205-weighting-of-atoms-in-composer) from 2017 one day, I’m sure…